From 3dd3ae46c1a8bdb1bdb7d08a76cdfe447e0bae2e Mon Sep 17 00:00:00 2001
From: Sergio Gonzalez Monroy <sgmonroy@gmail.com>
Date: Tue, 9 Mar 2021 22:47:51 +0100
Subject: [PATCH] linux-cp: make hidden symbols public

Move functions currently hidden to liblcp, thus making them public.

Type: fix

Change-Id: Iaaf578acf1331f3b5b813beb39a8d33b592c0e75
Signed-off-by: Sergio Gonzalez Monroy <sgmonroy@gmail.com>
---
 src/plugins/linux-cp/lcp_api.c       | 16 ----------------
 src/plugins/linux-cp/lcp_interface.c | 16 ++++++++++++++++
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/src/plugins/linux-cp/lcp_api.c b/src/plugins/linux-cp/lcp_api.c
index e5646732e..ed88acea8 100644
--- a/src/plugins/linux-cp/lcp_api.c
+++ b/src/plugins/linux-cp/lcp_api.c
@@ -41,22 +41,6 @@ api_encode_host_type (lip_host_type_t type)
   return LCP_API_ITF_HOST_TAP;
 }
 
-void
-lcp_set_auto_intf (u8 is_auto)
-{
-  lcp_main_t *lcpm = &lcp_main;
-
-  lcpm->auto_intf = (is_auto != 0);
-}
-
-int
-lcp_auto_intf (void)
-{
-  lcp_main_t *lcpm = &lcp_main;
-
-  return lcpm->auto_intf;
-}
-
 static void
 vl_api_lcp_itf_pair_add_del_t_handler (vl_api_lcp_itf_pair_add_del_t *mp)
 {
diff --git a/src/plugins/linux-cp/lcp_interface.c b/src/plugins/linux-cp/lcp_interface.c
index 534d974c7..eddffb365 100644
--- a/src/plugins/linux-cp/lcp_interface.c
+++ b/src/plugins/linux-cp/lcp_interface.c
@@ -60,6 +60,22 @@ static uword *lip_db_by_vif;
 index_t *lip_db_by_phy;
 u32 *lip_db_by_host;
 
+void
+lcp_set_auto_intf (u8 is_auto)
+{
+  lcp_main_t *lcpm = &lcp_main;
+
+  lcpm->auto_intf = (is_auto != 0);
+}
+
+int
+lcp_auto_intf (void)
+{
+  lcp_main_t *lcpm = &lcp_main;
+
+  return lcpm->auto_intf;
+}
+
 #define LCP_ITF_PAIR_DBG(...)                                                 \
   vlib_log_notice (lcp_itf_pair_logger, __VA_ARGS__);
 
-- 
2.17.1

